src/components/managed-street.js

```diff
@@ -5,14 +5,137 @@ const { segmentVariants } = require('../segments-variants.js');
 const streetmixUtils = require('../tested/streetmix-utils');
 const streetmixParsersTested = require('../tested/aframe-streetmix-parsers-tested');
 
-// invoking from js console
+// invoking from js console - load from streetmix
 /*
 userLayersEl = document.getElementById('street-container');
 newStreetEl = document.createElement('a-entity');
-newStreetEl.setAttribute('managed-street', 'sourceType: streetmix-url; sourceValue: https://streetmix.net/kfarr/3/; synchronize: true');
+newStreetEl.setAttribute('managed-street', {
+  sourceType: 'streetmix-url',
+  sourceValue: 'https://streetmix.net/kfarr/3/',
+  synchronize: true
+});
+userLayersEl.append(newStreetEl);
+*/
+
+// invoking from js console - load from blob
+/*
+userLayersEl = document.getElementById('street-container');
+newStreetEl = document.createElement('a-entity');
+newStreetEl.setAttribute('managed-street', {
+  sourceType: 'json-blob',
+  sourceValue: window.STREET.SAMPLE_STREET,
+  synchronize: true
+});
 userLayersEl.append(newStreetEl);
 */
 
+// Example Street object
+// 'width' this is the user-specified RoW width, not cumulative width of segments
+// some of this is redundant, many of the segment attributes defined below can be inferred by type
+window.STREET.SAMPLE_STREET = {
```

Comment:
what's the purpose of having this and is there a way to have this outside of this file to keep it clean?

Reply:
it's for testing and yeah it should be somewhere outside of this file

Reply:
ok moved example scenes out of this file
https://github.com/3DStreet/3dstreet/pull/965/commits/cee039a1504ab40fa5b4a842afa57d64adb15c65